src/portable/st/stm32_fsdev/dcd_stm32_fsdev.c

```diff
@@ -203,6 +203,11 @@ static inline void reg16_clear_bits(__IO uint16_t *reg, uint16_t mask) {
   *reg = (uint16_t)(*reg & ~mask);
 }
 
+// Bits in ISTR are cleared upon writing 0
+static inline void reg16_clear_istr(__IO uint16_t *reg, uint16_t mask) {
```

Comment:
Function name and comment suggests that it will work on **ISTR** register.
There is not point in passing register address as argument in my opinion.
I would also get rid of *reg16_*, it was OK when generic register was modified.
Just opinion.

Reply:
Noted. I've updated the PR.